frappe/gettext/extractors/javascript.py

```diff
@@ -0,0 +1,26 @@
+from babel.messages.extract import extract_javascript
+
+
+def extract(fileobj, keywords, comment_tags, options):
```

Comment:
@barredterra AFAIK extract_javascript from babel doesn't work for us. Hence I copied it and modified it to work with Frappe codebase. 

You can find it here: https://github.com/frappe/frappe/blob/7d4bdbebb9c80742e098b97bb1008fdb8c340fcc/frappe/translate.py#L678-L701

It will to again be modified (or just wrapped) for it to work with babel interface.

Reply:
If all of this is somehow handled then let me know :eyes: